rclrs/src/context.rs

```diff
@@ -118,6 +127,47 @@ impl Context {
     }
 }
 
+/// Returns non-ROS arguments held by context.
+///
+/// This function should be called after rcl_init. `rcl_arguments` should be `global_arguments`
+/// field of initialized context. `args` is array of input arguments passed to node.
+fn get_non_ros_arguments(
+    rcl_arguments: *const rcl_arguments_t,
+    args: &[*const c_char],
+) -> Result<Vec<String>, RclrsError> {
+    // SAFETY: In case of error -1 will be returned and handled a line below.
```

Comment:
The purpose of a safety comment (in my mind) is to indicate what things need to be done in order to prevent undefined behavior (UB) from occurring from the result of running an `unsafe` function, or from using its output. In other words, what needs to be done to keep the `unsafe` function from causing UB, or what needs to be done to safely use its output.

Since returning a `-1` will not cause UB, and because using/accessing a -1 is safe, I believe that this should be:

```suggestion
    // SAFETY: No preconditions for this function.
```

Reply:
Oh right, seems that I misunderstood the concept of `SAFETY` comments. Indeed `rcl_arguments_get_count_unparsed` needs no precondition, since it's so foolproof, that it can deal with `NULL` also.